### PR TITLE
Remove myget

### PR DESCRIPTION
**Pull request by live1206**
[AB#223304](https://dev.azure.com/ceapex/Engineering/_workitems/edit/223304)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5947)

### CODE REVIEW

File: azure-pipelines-v3.yml [hunk 1]
```diff
@@ -75,19 +75,19 @@ jobs:
...
+  # Push to sandbox vsts feed
   - task: NuGetCommand@2
     condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/v3'))
     displayName: Push to Sandbox MyGet Feed
     inputs:
       command: push
       packagesToPush: drop/**/*.nupkg
-      nuGetFeedType: external
-      publishFeedCredentials: myget.docfx-v3-sandbox
+      publishVstsFeed: 'docfx/docs-build-v3-test'
```
Comment (yufeih) [inline azure-pipelines-v3.yml:85]: We tend to put all public packages to `docs-public-packages` feed: `<add key="doc-public-packages" value="https://docfx.pkgs.visualstudio.com/docfx/_packaging/docs-public-packages/nuget/v3/index.json" />`

Also update README.md to point to the new feed

File: azure-pipelines-v3.yml [hunk 2]
```diff
@@ -75,19 +75,19 @@ jobs:
       DOCS_OPS_TOKEN: $(DOCS_OPS_TOKEN)
       CODECOV_TOKEN: $(CODECOV_TOKEN)
 
-  # Push to sandbox MyGet feed
+  # Push to sandbox vsts feed
```
Comment (yufeih) [inline azure-pipelines-v3.yml:78]:
```suggestion
  # Push to sandbox azure devops feed
```

File: azure-pipelines-v3.yml [hunk 3]
```diff
@@ -75,19 +75,19 @@ jobs:
...
     displayName: Push to Sandbox MyGet Feed
     inputs:
       command: push
       packagesToPush: drop/**/*.nupkg
-      nuGetFeedType: external
-      publishFeedCredentials: myget.docfx-v3-sandbox
+      publishVstsFeed: 'docfx/docs-build-v3-test'
+      allowPackageConflicts: true
 
   # Push to production MyGet feed
```
Comment (yufeih) [inline azure-pipelines-v3.yml:88]: It this still using myget?

File: azure-pipelines-v3.yml [hunk 4]
```diff
@@ -75,25 +75,15 @@ jobs:
...
 
-  # Push to sandbox MyGet feed
+  # Push to sandbox azure devops feed
   - task: NuGetCommand@2
     condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/v3'))
     displayName: Push to Sandbox MyGet Feed
     inputs:
       command: push
       packagesToPush: drop/**/*.nupkg
-      nuGetFeedType: external
```
Comment (herohua) [inline azure-pipelines-v3.yml:85]: I thought we should push to other two feeds - would you take a look at the description in the item?

Comment (live1206) [reply azure-pipelines-v3.yml:85]: Please check my comment in the work item

**Pull request merged**